pkg/util/node/node_test.go

```diff
@@ -94,110 +91,6 @@ func TestGetPreferredAddress(t *testing.T) {
 	}
 }
 
-func TestGetNodeHostIPs(t *testing.T) {
-	testcases := []struct {
-		name      string
-		addresses []v1.NodeAddress
-
-		expectIPs []net.IP
-	}{
-		{
-			name:      "no addresses",
-			expectIPs: nil,
-		},
-		{
-			name: "no InternalIP/ExternalIP",
-			addresses: []v1.NodeAddress{
-				{Type: v1.NodeHostName, Address: "example.com"},
-			},
-			expectIPs: nil,
-		},
-		{
-			name: "IPv4-only, simple",
-			addresses: []v1.NodeAddress{
-				{Type: v1.NodeInternalIP, Address: "1.2.3.4"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.1"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.2"},
-			},
-			expectIPs: []net.IP{netutils.ParseIPSloppy("1.2.3.4")},
-		},
-		{
-			name: "IPv4-only, external-first",
-			addresses: []v1.NodeAddress{
-				{Type: v1.NodeExternalIP, Address: "4.3.2.1"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.2"},
-				{Type: v1.NodeInternalIP, Address: "1.2.3.4"},
-			},
-			expectIPs: []net.IP{netutils.ParseIPSloppy("1.2.3.4")},
-		},
-		{
-			name: "IPv4-only, no internal",
-			addresses: []v1.NodeAddress{
-				{Type: v1.NodeExternalIP, Address: "4.3.2.1"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.2"},
-			},
-			expectIPs: []net.IP{netutils.ParseIPSloppy("4.3.2.1")},
-		},
-		{
-			name: "dual-stack node",
-			addresses: []v1.NodeAddress{
-				{Type: v1.NodeInternalIP, Address: "1.2.3.4"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.1"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.2"},
-				{Type: v1.NodeInternalIP, Address: "a:b::c:d"},
-				{Type: v1.NodeExternalIP, Address: "d:c::b:a"},
-			},
-			expectIPs: []net.IP{netutils.ParseIPSloppy("1.2.3.4"), netutils.ParseIPSloppy("a:b::c:d")},
-		},
-		{
-			name: "dual-stack node, different order",
-			addresses: []v1.NodeAddress{
-				{Type: v1.NodeInternalIP, Address: "1.2.3.4"},
-				{Type: v1.NodeInternalIP, Address: "a:b::c:d"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.1"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.2"},
-				{Type: v1.NodeExternalIP, Address: "d:c::b:a"},
-			},
-			expectIPs: []net.IP{netutils.ParseIPSloppy("1.2.3.4"), netutils.ParseIPSloppy("a:b::c:d")},
-		},
-		{
-			name: "dual-stack node, IPv6-first, no internal IPv4, dual-stack cluster",
-			addresses: []v1.NodeAddress{
-				{Type: v1.NodeInternalIP, Address: "a:b::c:d"},
-				{Type: v1.NodeExternalIP, Address: "d:c::b:a"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.1"},
-				{Type: v1.NodeExternalIP, Address: "4.3.2.2"},
-			},
-			expectIPs: []net.IP{netutils.ParseIPSloppy("a:b::c:d"), netutils.ParseIPSloppy("4.3.2.1")},
-		},
-	}
-
-	for _, tc := range testcases {
-		t.Run(tc.name, func(t *testing.T) {
-			node := &v1.Node{
-				Status: v1.NodeStatus{Addresses: tc.addresses},
-			}
-			nodeIPs, err := GetNodeHostIPs(node)
```

Comment:
`GetNodeHostIPs` is still in used, so I don't believe this test case should be removed. correct me if I'm wrong.